tests/com/splunk/PasswordTest.java

```diff
@@ -53,14 +53,12 @@ public void testPasswordCrud() {
         password = passwords.create(name, value);
         Assert.assertTrue(passwords.containsKey(name));
         Assert.assertEquals(name, password.getUsername());
-        Assert.assertEquals(value, password.getClearPassword());
```

Comment:
Instead of removing the assertion, we should check for a thrown exception (due to nonexisting underlying `clear_password` value).
The test should also check that the current Splunk version is at least the version where that field is no longer shown in the body of POST requests